x-pack/plugins/ml/public/application/components/data_grid/use_column_chart.ts

```diff
@@ -0,0 +1,336 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import { BehaviorSubject } from 'rxjs';
+
+import { useObservable } from 'react-use';
+
+import { euiPaletteColorBlind, EuiDataGridColumn } from '@elastic/eui';
+
+import { KBN_FIELD_TYPES } from '../../../../../../../src/plugins/data/public';
+
+import { stringHash } from '../../../../common/util/string_utils';
+
+import { NON_AGGREGATABLE } from './common';
+
+export const hoveredRow$ = new BehaviorSubject<any | null>(null);
+
+const BAR_COLOR = euiPaletteColorBlind()[0];
+const BAR_COLOR_BLUR = euiPaletteColorBlind(2)[10];
+const MAX_CHART_COLUMNS = 20;
+
+const getXScaleType = (
+  kbnFieldType: KBN_FIELD_TYPES
+): 'ordinal' | 'time' | 'linear' | undefined => {
+  switch (kbnFieldType) {
+    case KBN_FIELD_TYPES.BOOLEAN:
+    case KBN_FIELD_TYPES.IP:
+    case KBN_FIELD_TYPES.STRING:
+      return 'ordinal';
+    case KBN_FIELD_TYPES.DATE:
+      return 'time';
+    case KBN_FIELD_TYPES.NUMBER:
+      return 'linear';
+  }
+};
+
+const getFieldType = (schema: EuiDataGridColumn['schema']): KBN_FIELD_TYPES | undefined => {
+  if (schema === NON_AGGREGATABLE) {
+    return undefined;
+  }
+
+  let fieldType: KBN_FIELD_TYPES;
+
+  switch (schema) {
```

Comment:
I'm getting this error from the `iowa_liquor_sales` data. One of the fields has type `geo_point` - not sure if that is the one causing the page to hang:

![image](https://user-images.githubusercontent.com/7405507/84651872-8ab2f200-af02-11ea-9849-e2774661275c.png)

Reply:
Changed the way unsupported fields get treated in 3393f59, please have another look with the `iowa` and `cloudwatch` dataset.

Reply:
Working for me too with latest changes with the `iowa_liquor_sales` and `cloudwatch` data sets!